code/modules/mob/living/simple_animal/vore/solargrub.dm

```diff
@@ -78,7 +78,7 @@ List of things solar grubs should be able to do:
 			anchored = 1
 			PN = attached.powernet
 			PN.draw_power(150000)
-			var/apc_drain_rate = 4000 //5000 seems a bit high, knocking it down to 4000
+			var/apc_drain_rate = 750 //5000 seems a bit high, knocking it down to 4000
```

Comment:
-"5000 seems a bit high, knocking it down to 4000"
Should update this line.

Reply:
True. Also not lowering the 150000? Want that rate to stay the same, or?

Reply:
That draws power directly from the powernet, doesn't affect it that much. Apc drain rate is much worse? I'll knock it down a bit anyways.